designer/client/cypress/e2e/fragment.cy.ts

```diff
@@ -299,9 +299,6 @@ describe("Fragment", () => {
 
         cy.get("[data-testid=window]").find("section").scrollTo("top");
         cy.get("[data-testid=window]").find('[data-testid="content-size"]').matchImage();
-
-        cy.get("[data-testid=window]").find("section").scrollTo("bottom");
-        cy.get("[data-testid=window]").find('[id="nk-graph-fragment"]').matchImage();
```

Comment:
This corresponds to `Fragment should allow adding input parameters and display used fragment graph in modal #9.png`  file - we can delete it now?